src/index.html

```diff
@@ -77,6 +77,10 @@
       </symbol>
 
       <!-- Dataflow icons TODO: move to plugins/dataflow-tool folder -->
+      <symbol id="icon-dropdown-caret" viewBox="0 0 7 5">
+        <title>dropdown caret</title>
+        <polygon id="dropdown_caret-2" data-name="dropdown caret" points="3.5 5 0 1.41 1.44 0 3.5 2.1 5.56 0 7 1.41 3.5 5"/>
+      </symbol>
```

Comment:
Is it possible to _not_ keep adding more SVGs to the `index.html`? We have standard mechanisms for handling SVGs by importing them directly as React components. Is there a particular reason we can't do that with these?

Reply:
275 of them by my count 😉 -- run this as a multi-file regex search in VSCode: `import .* from ".*\.svg"`.

The short version is to just put the .svg file in an appropriate place (e.g. `assets/icons`) and then `import` it into the module/component in which you'd like to use it. Under the hood the `@SVGR` webpack plugin converts it into a React component that can be used like any other component.

Reply:
For Dataflow-specific icons, that "appropriate place" should be within the dataflow folder. Look in the diagram or drawing tiles for conventions on how to arrange assets in "plugin" tiles.